introduction.md

```diff
@@ -1,3 +1,13 @@
 hello my name is Jimmy and I don't like introductions
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?